django_project/static/css/style.css

```diff
@@ -1907,4 +1907,3 @@
      font-weight: 500;
      text-align: center;
 }
- 
```

Comment:
static and media folders should be in git ignore. And remove these folders from the project